src/content/ru/2021/seo.md

```diff
@@ -0,0 +1,758 @@
+   caption="Использование заголовка `Vary: User-Agent`.",
+   description="Гистограмма показывает использование заголовка `vary`, необходимого для распознавания удобства для мобильных устройств. Мы обнаружили, что адаптивный дизайн на страницах используется чаще (87.4% для компьютеров и 86.6% для мобильных устройств), чем динамический показ (12.6% для компьютеров и 13.4% для мобильных устройств).",
+   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQ7oSHa6DHYTfZqsuGmIYdlfYVkwrUvOYD_r6soecExV_ZpbbZjmG6watu0hwrOKqK3-inNrt0TfXCO/pubchart?oid=1985736287&format=interactive",
+   sheets_gid="478009067",
+   sql_file="html-response-vary-header-used.sql"
+   )
+}}
+
+Для заголовка `Vary: User-Agent` цифры почти не изменились: 12.6% страниц для компьютеров и 13.4% мобильных страниц.
+
+{{ figure_markup(
+  caption="Процент мобильных страниц, не использующих разборчивые размеры шрифта.",
+  content="13.5%",
+  classes="big-number",
+  sheets_gid="1705330480",
+  sql_file="lighthouse-seo-stats.sql"
+)
+}}
+
+Одна из главных причин неудобства страниц для мобильных устройств — 13.5% страниц не используют разборчивые размеры шрифта. Это значит, что <a hreflang="en" href="https://web.dev/font-size/">60% или больше текста на странице имеют размер шрифта меньше 12px</a>, что может сложно читаться на мобильных устройствах. 
+
+### Core Web Vitals
+
+Core Web Vitals — это новые показатели скорости, которые являются частью сигналов удобства страниц для Google. Метрики измеряют производительность загрузки с помощью Largest Contentful Paint (LCP), визуальную стабильность с Cumulative Layout Shift (CLS) и интерактивность с First Input Delay (FID).
+
+Данные поступают из отчета о пользовательском опыте Chrome (CrUX), в котором записываются реальные данные от зарегистрированных пользователей Chrome.
+
+{{ figure_markup(
+   image="core-web-vitals-trend.png",
+   caption="Тенденция показателей Core Web Vitals.",
+   description="Линейная диаграмма показывает процент веб-сайтов с хорошими показателями Core Web Vitals на мобильных устройствах. В 2021 году доля сайтов с хорошим показателем LCP выросла с 42% до 45%, с хорошим FID — с 81% до 90%, с хорошим CLS — с 55% до 62%, и в целом с хорошими данными CWV — с 23% до 29%. Наши результаты показывают, что процент мобильных веб-сайтов с хорошими показателями Core Web Vitals будет продолжать расти с каждым годом.",
+   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQ7oSHa6DHYTfZqsuGmIYdlfYVkwrUvOYD_r6soecExV_ZpbbZjmG6watu0hwrOKqK3-inNrt0TfXCO/pubchart?oid=1693723951&format=interactive",
+   sheets_gid="460991760",
+   sql_file="core-web-vitals.sql"
+   )
+}}
+
+29% мобильных веб-сайтов удовлетворяют пороговым значениям Core Web Vitals по сравнению с 20% в прошлом году. У большинства веб-сайтов удовлетворительный показатель FID, но владельцы веб-сайтов, похоже, пытаются улучшить CLS и LCP. Дополнительную информацию по этой теме смотрите в главе [Производительность](./performance).
+
+## On-Page
```

Comment:
Should this not be translated? Or do you consider this a special technical term?

Reply:
If you are talking about `Core Web Vitals`, actually it can be translated. And even [here](https://developers.google.com/search/docs/advanced/experience/page-experience?hl=ru#signals) in the documentation there is a translation. But for example [here](https://web.dev/vitals/#core-web-vitals) and [here](https://web.dev/defining-core-web-vitals-thresholds/) it is not. In a Russian society of SEO specialists, translation is extremely rarely used, in 99% of cases they use the English name of the term.

Reply:
I was actually talking about "On-Page"? I agree CWV is definitely a technical term (if not a brand name) - neither of which needs to be translated, so fine with this. But "on-page" I'm not so sure about. WDYT?

Reply:
In the Russian SEO community, the English term is used more often. For example, [here](https://ahrefs.com/blog/ru/on-page-seo/). But unlike CWV, the translation of the term is still sometimes used. I would choose the English term, referring to the article Ahrefs.

Reply:
Sounds good to me. Just thought I'd check.